Makefile

```diff
@@ -22,13 +17,10 @@ test: venv
 	venv/bin/tox
 
 test-unit: venv
-	venv/bin/tox -e $(TOX_ENVLIST) -- $(TOX_ARGS)
+	venv/bin/tox -e py3
```

Comment:
I don't mind this change, but I find useful to have the TOX_ARGS variable here, as it allows us to filter which tests to execute:
```
TOX_ARGS="-k pip" make test-unit
```

I always missed it in the `test-integration` command, btw.

Reply:
On the other hand, I think I've been mostly calling tox directly, so if nobody else finds it uself, let's drop it.

Reply:
Personally, I use `tox -e py3 -- -k pip`.